src/components/input-file-snapshot.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useTranslator } from '../hooks';
+
+import {
+  FormControl,
+  InputLabel,
+  OutlinedInput,
+  FormHelperText
+} from '@mui/material';
+
+export const InputFileSnapshot = (props: {
+  inputFileSnapshot: string;
+}): JSX.Element => {
+  const trans = useTranslator('jupyterlab');
+
+  return (
+    <FormControl sx={{ border: 'undefined' }}>
```

Comment:
`undefined` is not a valid value for `border` in CSS. If you need to explicitly specify "no border", use `unset`. See docs: https://developer.mozilla.org/en-US/docs/Web/CSS/border

Reply:
`border: 'unset'` works well and is standard-compliant. Thank you!